bindings/rust/extended/s2n-tls/src/config.rs

```diff
@@ -1163,4 +1182,52 @@ mod tests {
 
         Ok(())
     }
+
+    #[cfg(all(
+        // The `add_custom_x509_extension` API is only exposed when its unstable feature is enabled.
+        feature = "unstable-custom_x509_extensions",
+        // The `add_custom_x509_extension` API is only supported with AWS-LC, so
+        // this test is disabled for the external build, which may link to other libcryptos.
+        not(s2n_tls_external_build),
+        // The `add_custom_x509_extension` API is currently unsupported with AWS-LC-FIPS.
```

Comment:
Hm, what is the blocker to supporting this with FIPS? Is that a compliance problem or a timeline issue (we don't have awc-lc-fips-sys released with the APIs?)